modules/optimization/doc/content/modules/optimization/sqa/optimization_sdd.md

```diff
@@ -1 +1,60 @@
-!template load file=app_sdd.md.template app=Optimization category=optimization
+!template load file=sqa/module_sdd.md.template app=Optimization category=optimization
+
+!template! item key=introduction
+The [!ac](MOOSE) {{module}} module is based on the MOOSE framework and thus inherits
+the unique features and base characteristics of the framework, as outlined in the [framework_sdd.md].
+Specific details unique to the module are outlined in this document.
+!template-end!
+
+!template! item key=system-scope
+!include optimization_srs.md start=system-scope-begin end=system-scope-finish
+!template-end!
+
+!template! item key=dependencies-and-limitations
+The {{module}} module inherits the
+[software dependencies of the MOOSE framework](framework_sdd.md#dependencies-and-limitations),
+with no additional dependencies. The module relies heavily on the TAO optimization package, which is available through the framework's inclusion of the PETSc. The design of this module is motivated by the
+needs of its client applications.
+!template-end!
+
+!template! item key=design-stakeholders
+!include framework_sdd.md start=design-stakeholders-begin end=design-stakeholders-finish
+!template-end!
+
+!template! item key=system-design
+The {{module}} module inherits the wide range of pluggable systems from MOOSE.
+More information regarding MOOSE system design can be found in the framework
+[framework_sdd.md#system-design] section. Most of these inherited systems are
+utilized to build physics models for which the optimization is being
+performed for. Other than these, the MultiApps, Transfers, and Reporters systems
```

Comment:
```suggestion
utilized to build physics models for which optimization is being
performed. Other than these, the MultiApps, Transfers, and Reporters systems
```

Also might want to link to these system landing pages, but that's optional to me.